cachi2/core/package_managers/yarn_classic/main.py

```diff
@@ -91,11 +90,55 @@ def _generate_build_environment_variables() -> list[EnvironmentVariable]:
     return [EnvironmentVariable(name=key, value=value) for key, value in env_vars.items()]
 
 
+def _check_pnp_installs(project: Project) -> None:
+    if project.is_pnp_install:
+        raise PackageRejected(
+            reason=("Yarn zero install detected, PnP zero installs are unsupported by cachi2"),
```

Comment:
I think "zero-installs" are a term/feature that only really exists in Yarn v2+, so we might want to avoid using it here for Yarn 1.x. Same with the solution text mention of .yarn/cache

Reply:
Gah. Copy-pasta.

Reply:
Fixed